pkg/kv/kvserver/batcheval/cmd_export_test.go

```diff
@@ -55,14 +55,26 @@ func TestExportCmd(t *testing.T) {
 	dir, dirCleanupFn := testutils.TempDir(t)
 	defer dirCleanupFn()
 	srv, db, kvDB := serverutils.StartServer(t, base.TestServerArgs{
-		DefaultTestTenant: base.TestIsForStuffThatShouldWorkWithSecondaryTenantsButDoesntYet(109429),
-		ExternalIODir:     dir,
+		ExternalIODir: dir,
 	})
 	defer srv.Stopper().Stop(ctx)
 	ts := srv.ApplicationLayer()
 
+	sysDB := sqlutils.MakeSQLRunner(srv.SystemLayer().SQLConn(t, ""))
+
 	sql.SecondaryTenantSplitAtEnabled.Override(ctx, &ts.ClusterSettings().SV, true)
 
+	// sKey takes a string key and makes a key, adding a tenant prefix if needed.
+	sKey := func(s string) roachpb.Key {
+		if ts.Codec().ForSystemTenant() {
+			return []byte(s)
+		}
+		if s == keys.MaxKey.String() {
+			return ts.Codec().TenantSpan().EndKey
+		}
+		return []byte(ts.Codec().TenantSpan().Key.String() + s)
```

Comment:
I find this more readable: `return append(ts.Codec().TenantPrefix(), s...)`

(also what i've used in other places)